src/config/locales.md

```diff
@@ -12,9 +17,10 @@ To enable a certain locale, un-comment or add the relevant lines in
 `/etc/default/libc-locales` and [force-reconfigure](../xbps/index.md) the
 `glibc-locales` package.
 
-## Setting the system locale
+## Setting the system language
 
-Set `LANG=xxxx` in `/etc/locale.conf`.
+Set `LANG=xxxx` in `/etc/locale.conf`, this applies to `glibc` and `musl`
+although it's not required by the latter.
```

Comment:
it is required for glibc, if you want the language to change for most applications

Reply:
I never said it wasn't though, did you mean to say musl instead of glibc?

Reply:
i read latter as former, but `LANG` is required for setting the system language on both libcs

Reply:
I do that too sometimes, I'll remove that part altogether.

Reply:
Okay, done.